lib/modules/manager/composer/extract.ts

```diff
@@ -44,6 +44,13 @@ function parseRepositories(
 
         switch (repo.type) {
           case 'vcs':
+          case 'bitbucket':
```

Comment:
The documentation for this method should also be updated. Currently it only mentions `vcs` and `git`.